package.json

```diff
@@ -49,29 +50,24 @@
     "@types/deep-equal-in-any-order": "^1.0.1",
     "@types/expect": "^24.3.0",
     "@types/form-data": "^2.5.0",
-    "@types/lodash.isequal": "^4.5.5",
     "@types/mocha": "^9.1.1",
     "@types/node": "^17.0.0",
     "@types/uuid": "^8.3.3",
     "@typescript-eslint/eslint-plugin": "^5.2.0",
     "@typescript-eslint/parser": "^5.2.0",
     "chai": "^4.3.6",
+    "deep-equal": "^2.0.5",
```

Comment:
I don't think we need this *in* the build, so dev deps should be fine (only used in tests)